firebase_stream_block.py

```diff
@@ -0,0 +1,26 @@
+from nio.util.discovery import discoverable
+from nio.properties import VersionProperty
+from nio.signal.base import Signal
+
+from .firebase_base import FirebaseBase
+
+
+@discoverable
+class FirebaseStream(FirebaseBase):
+
+    version = VersionProperty("2.0.0")
```

Comment:
If this is a new block I would initialize the version as 1.0.0. Don't forget about the spec and release versions for this block as well.